app/services/import/legacy/fetch-company.service.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+/**
+ * Fetches the company data from the import.NALD_PARTIES table for the licence ref
+ * @module FetchCompanyService
+ */
+
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Fetches the licence data from the import.NALD_PARTIES table for the licence ref
+ *
+ * After this point parties will be referred to as companies
+ *
+ * @param {string} regionCode - The NALD region code
+ * @param {string} licenceId - The NALD licence ID
+ *
+ * @returns {Promise<ImportLegacyCompaniesType[]>}
+ */
+async function go (regionCode, licenceId) {
+  const query = _query()
+
+  const { rows } = await db.raw(query, [regionCode, licenceId])
+
+  return rows
+}
+
+function _query () {
```

Comment:
I think we actually need to get the parties from the lic version as well here. 

I can see locally that there is a party in the licence version table that is not in the licence roles table. ( the licence holder as we see it in the UI).

I might add it as another query have the transform query handle merging together the companies for persisting  and try and return the same data object shape to create the contacts and address from the `ACON_AADD_ID` and `ACON_APAR_ID`